src/hooks/usePlaceOrder.ts

```diff
@@ -158,9 +163,9 @@ export const usePlaceOrder = (): Result => {
           sellTokens.push(order.sellToken)
 
           // if not set, order is valid from placement + wait period
-          validFroms.push(order.validFrom ?? currentBatchId + BATCHES_TO_WAIT)
+          validFroms.push(order.validFrom ? currentBatchId + order.validFrom : currentBatchId + BATCHES_TO_WAIT)
```

Comment:
```suggestion
          validFroms.push(currentBatchId + order.validFrom ? order.validFrom : BATCHES_TO_WAIT)
```

Reply:
wait, wait, these aren't equivalent
```suggestion
          validFroms.push(currentBatchId + (order.validFrom ? order.validFrom : BATCHES_TO_WAIT))

```

Reply:
oops, pretend you didn't see that